src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Certificates/StartCertificateHandlerTests/When_called_and_no_existing_certificate.cs

```diff
@@ -95,6 +115,18 @@ public void Arrange()
                         Username = "user"
                     }, new CancellationToken()).Wait();
 
+            _startCertificateHandler
+                .Handle(
+                    new StartCertificateRequest()
+                    {
+                        StandardCode = 30,
+                        UkPrn = 88888888,
+                        Uln = 333333333,
+                        Username = "user",
+                        CourseOption = "CourseOption",
+                        StandardUId = "ST0016",
+                    }, new CancellationToken()).Wait();
+
             _returnedCertificate = _startCertificateHandler
                 .Handle(
```

Comment:
Should these Handler calls be in their own specific tests rather than 3 in the setup? Also, the last call assigns the certificate to the `_returnedCertificate` field but the field isn't being checked against it just Verifies the repository method.

Reply:
I was holding off on reworking the test class but it's probably the right time to do it. I don't like the way it's structured so will change it to how we've upgraded them elsewhere. Watch this space.

Reply:
I have removed the horrible set of tests and then added in a new file that's hopefully neater